css/block.css

```diff
@@ -156,6 +158,84 @@ wb-context[closed] > wb-contains{
     overflow: hidden;
 }
 
+
+wb-value[type=number] > input{
```

Comment:
I think this can be simplified, actually. All your changer per each `wb-value[type=blah]` is the `background-image`, right?

So you can set all the other stuff (i.e., `no-repeat scroll 3px 3px : #FFFFFF`) in just one rule, for all `wb-value > input` using the [background shorthand](http://mdn.io/background-shorthand). Then, for each of these rules, just set the `background-image`. This should cut down on code duplication.